src/core/stdc/stdio.d

```diff
@@ -1516,9 +1520,9 @@ else version (OpenBSD)
     enum __SALC = 0x4000;
     enum __SIGN = 0x8000;
 
-    extern int __isthreaded;
+    extern immutable __gshared int __isthreaded;
 
-    extern (D)
+    extern (D) @trusted
```

Comment:
Why would these not be `@safe` already?

Reply:
It's a copy-pasto from another block. Ideally they'd all be commonized and there be only one place for the definition/signature.